radicle-httpd/src/api/v1/projects.rs

```diff
@@ -349,7 +348,7 @@ async fn issues_handler(
 /// `GET /projects/:project/issues/:id`
 async fn issue_handler(
     Extension(ctx): Extension<Context>,
-    Path((project, issue_id)): Path<(Id, Oid)>,
+    Path((project, issue_id)): Path<(Id, radicle::git::Oid)>,
```

Comment:
Why can't they be substituted?

Reply:
For some reason, rustc / cargo didn't like it.  For example, if I removed `readicle::git::` here, I would get the following error:

```
error[E0277]: the trait bound `ObjectId: From<radicle_surf::git::Oid>` is not satisfied
   --> radicle-httpd/src/api/v1/projects.rs:356:15
    |
356 |         .get(&issue_id.into())?
    |               ^^^^^^^^ ---- required by a bound introduced by this call
    |               |
    |               the trait `From<radicle_surf::git::Oid>` is not implemented for `ObjectId`
    |
    = help: the following other types implement trait `From<T>`:
              <ObjectId as From<&radicle::git::Oid>>
              <ObjectId as From<&radicle::git::git2::Oid>>
              <ObjectId as From<radicle::git::Oid>>
              <ObjectId as From<radicle::git::git2::Oid>>
    = note: required for `radicle_surf::git::Oid` to implement `Into<ObjectId>`
```

Reply:
There may be two versions imported somehow, I'll take a look.

Reply:
Looks like it was because the root `Cargo.toml` has a patch section for `radicle-git-ext`. I've updated the diff to use the same version for the patch. Now the problem is gone.